packages/client/src/components/Dashboard/NavigationBar.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Typography } from '@mui/material';
+import HomeOutlinedIcon from '@mui/icons-material/HomeOutlined';
+import HelpOutlineOutlinedIcon from '@mui/icons-material/HelpOutlineOutlined';
+import DescriptionOutlinedIcon from '@mui/icons-material/DescriptionOutlined';
+import ExitToAppOutlinedIcon from '@mui/icons-material/ExitToAppOutlined';
+
+const navigationData = [
```

Comment:
There is no need for an extra abstaction here, we can just use the navigation items directly in the NavigationBar